pkg/terraformer/config.go

```diff
@@ -101,6 +101,18 @@ func (t *terraformer) InitializeWith(initializer Initializer) Terraformer {
 	return t
 }
 
+//GetState returns the contenten of terraform state config map
+func (t *terraformer) GetRawState(ctx context.Context) (*RawState, error) {
+	configMap := &corev1.ConfigMap{}
+	if err := t.client.Get(ctx, kutil.Key(t.namespace, t.stateName), configMap); err != nil {
+		return nil, err
+	}
+	return &RawState{
+		Data:   configMap.Data[StateKey],
+		Encode: "none",
```

Comment:
You could introduce constants for `none` and `base64`. Currently they are repeated multiple times.

Reply:
```suggestion
		Encode: NoneEncoding,
```